ipfs/ipfs-webui
# feat: remote pinning settings page

## Description

- Unstubs UI Elements to use Remote Pinning API (https://github.com/ipfs/js-ipfs/pull/3293)
- The CI is probably going to fail until we have a go-ipfs@8+ release and bump this PR to use it

## Testing

To test this locally, follow these steps: 
- Checkout js-ipfs into local dir and switch to @Gozala branch (feat/pin-remote)
- Go to js-ipfs/package/ipfs-http-client and try to build it via `npm ci`.
- Go back to ipfs-webui dir, and execute `connect-deps link /path/to/js-ipfs/packages/ipfs-http-client` and then `connect-deps connect`
if successful, you should be able to run `npm i && npm start` and have getIpfs().pin.remote
